<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestor</class>
 <widget class="QMainWindow" name="gestor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor FTP</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/img/LTakzdPkT5iXZdyYumu8uw.png</normaloff>:/img/LTakzdPkT5iXZdyYumu8uw.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>60</width>
    <height>60</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnStartServer">
        <property name="text">
         <string>Iniciar Servidor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStopServer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Detener Servidor</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabConsole">
       <attribute name="title">
        <string>Consola</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="txtConsoleOutput">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="txtCommandInput">
            <property name="placeholderText">
             <string>Ingrese comando (ej: start-server, stop-server, list-users)...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExecute">
            <property name="text">
             <string>Ejecutar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLogs">
       <attribute name="title">
        <string>Logs del Servidor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPlainTextEdit" name="txtLogs">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnLimpiarLogs">
            <property name="text">
             <string>Limpiar Logs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnGuardarLogs">
            <property name="text">
             <string>Guardar Logs</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
